### ISSUE TITLE
Run Appimages

### ISSUE THREAD

**Issue by JoshuaPettus** [ptitSeb/box64#55]
Maybe just a fever dream but then so is box64 :)
Is there anyway to run an x86_64 appimage?  when I try to run Musescore's app image

https://musescore.org/en/download

I get the standard
cannot execute binary file: Exec format error

I assume it has something to do with AppImage's contained nature not being picked up by box64

**Comment by JoshuaPettus** [comment]
Ah it's as simple as running it with the command box64 ./whatever.appimage
Perhaps something can be done with the autodetect.

But that just gets me to the door,  for some reason it says

> Error loading needed lib libfuse.so.2
> Warning: Cannot dlopen("libfuse.so.2"/0x421ac2, 1)
> dlopen(): error loading libfuse.so.2
> 
> AppImages require FUSE to run. 


Even though I have libfuse installed

> pi@raspberrypi:~/Downloads $ dpkg -l | grep libfuse
> ii  libfuse2:arm64                                              2.9.9-1+deb10u1                                           arm64        Filesystem in Userspace (library)
> ii  libfuse3-3:arm64                                            3.4.1-1+deb10u1                                           arm64        Filesystem in Userspace (library) (3.x version)
>

**Comment by Heasterian** [comment]
Extract it first and run extracted. Afaik, appimages are not supported now. Also, libfuse is not wrapped so you would need amd64 version (for example extracted from deb).

**Comment by JoshuaPettus** [comment]
Well part of the reason for this thread is to support them, but thanks.

I'm not following, once extract the amd64 version from a debian file, what do I do with it?

**Comment by Heasterian** [comment]
Place in directory with binary you want to start or in certain directory and use BOX64_LD_LIBRARY_PATH to define directory with amd64 libs.

**Comment by JoshuaPettus** [comment]
Ah ok!  I'll give that a shot

**Comment by JoshuaPettus** [comment]
Hmm dang, thanks for the advice.  Lib fuse was't needed as once I extracted the app image it wasn't an image anymore.  Which was fine.  Thing is I was hoping that with app images, that all the dependencies would already be contained in the app image.  But boy was I wrong.  Hours later traveling down the dependency rabbit hole installing in the appimage lib folder amd64 library after amd64 library, I ended up was a Segfault for my troubles.  Honestly not too surprising.  I should try a simpler program than Musescore.
